src/module.ts

```diff
@@ -190,6 +190,31 @@ export interface ModuleOptions {
   navigation: false | {
     fields: Array<string>
   }
+  /**
+   * Search mode.
+   *
+   * @default false
+   */
+  search: false | {
+    /**
+     * Used to determine how to search content.
+     */
+    mode: 'full-text'
+    /**
+     * List of tags where text must not be extracted. Only works with `full-text` mode.
+    *
+    * By default, will extract text from each tag.
+    *
+    * @default ['style', 'code']
+    */
+    ignoredTags?: Array<string>
+    /**
+     * API return indexed contents to improve client-side load time.
```

Comment:
Add the fact that using this, dev will need to use MiniSearch (or use search). Disabling this, and dev can use another system like fuse.js